### DISCUSSION TITLE
Pico W - Getting Started - picozero module not there?

### DISCUSSION THREAD

**Discussion by dave-ct** [micropython/micropython#9318] [Using MicroPython] [answered]
Tried the latest nightly build (rp2-pico-w-20220914-unstable-v1.19.1-409-g0e8c2204d.uf2) then following the guide at https://projects.raspberrypi.org/en/projects/get-started-pico-w/2 get the following error:
ImportError: no module named 'picozero'

Everyhting else works fine in terms of setting up the wifi on the picoW

Do you know if this is a bug in micropython build or is the reapberry pi documentation wrong (checking before logging an issue)?

**Comment by jimmo** [comment, marked as answer]
`picozero` isn't part of the base firmware, it's an optional thing you can install. See the previous page from that guide -- https://projects.raspberrypi.org/en/projects/get-started-pico-w/1

**Reply by dave-ct** [reply]
Thanks, was late a night and some how missed this……
